src/globus_cli/commands/group/set_policies.py

```diff
@@ -0,0 +1,109 @@
+from typing import Optional
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import CommaDelimitedList, command
+from globus_cli.termio import formatted_print
+
+from ._common import MEMBERSHIP_FIELDS, group_id_arg
+
+SIGNUP_FIELDS = [
+    "address",
+    "address1",
+    "address2",
+    "city",
+    "country",
+    "current_project_name",
+    "department",
+    "field_of_science",
+    "institution",
+    "phone",
+    "state",
+    "zip",
+]
+
+
+@click.option(
+    "--high-assurance/--not-high-assurance",
+    default=None,
+    help="Flag if the group will enforce high assurance access policies or not.",
```

Comment:
I think I might find this wording a little clearer?
```suggestion
    help="Whether the group should enforce high assurance access policies or not.",
```